doc/org.eclipse.cdt.doc.user/README.md

```diff
@@ -11,3 +11,43 @@ During content development, HTML files may be generated by invoking Maven locall
 ```
 mvn --define jgit.dirtyWorkingTree-cdtDefault=ignore --projects org.eclipse.cdt:org.eclipse.cdt.doc.user generate-resources
 ```
+## Embedding commands in help
+
+Embedding commands in help needs to modify syntax slightly.
+Links in adoc are always surrounded by double-quotes, so double-quotes in the link need to be escaped by use `&quot;`.
```

Comment:
True, but it's more readable to use single quotes where possible:
```
executeCommand('org.eclipse.ui.window.preferences(preferencePageId=org.eclipse.ui.preferencePages.Editors)')
```

Reply:
Yes! I'm so used to strings being `"` that I didn't even try that. Thanks for suggesting and I have made the updates.